apps/web/components/molecules/pagination/pagination.tsx

```diff
@@ -0,0 +1,107 @@
+import { ChevronLeftIcon, ChevronRightIcon } from '@heroicons/react/20/solid';
+import clsx from 'clsx';
+import { VisuallyHidden } from 'ui';
+import { useRouter } from 'next/router';
+
+type PaginationProps = {
+	quantity: number;
+};
+
+export const Pagination = ({ quantity }: PaginationProps) => {
+	const router = useRouter();
+	const currentPage: number =
+		router.query.page && !isNaN(Number(router.query.page))
+			? Number(router.query.page)
+			: 1;
+	const perPage =
+		router.query.page && !isNaN(Number(router.query.limit))
```

Comment:
shouldn't that be check for `router.query.limit`? Just asking coz I don' really know 😅 
Also I think it could be somehow extracted to some function as it is repeated two times here and kinda looks confusing even tho it just is some `validation/parsing/default` functionality.